### ISSUE TITLE
Google redirects for 18F case studies

### ISSUE THREAD

**Issue by cmajel** [18F/TLC-crew#391]
### A description of the work
(Please disregard if already covered by another ticket)

Recently, 18F case studies moved from https://18f.gsa.gov/our-work/ >>  https://18f.gsa.gov/what-we-deliver.

But, old urls still appear in Google and result in 404s. My examples:
* I googled `18f us tax court`, and the top result was this, which resulted in a 404: https://18f.gsa.gov/what-we-deliver/tax-court/
* I googled `18f fbi crime data explorer`,  and the top result was this, which resulted in a 404: https://18f.gsa.gov/what-we-deliver/crime-data-explorer/ .

I believe there are ways to [signal a redirect ](https://developers.google.com/search/docs/crawling-indexing/301-redirects#:~:text=When%20you%20redirect%20a%20URL,redirect%20was%20temporary%20or%20permanent.)to Google. I also wonder about places where we may be linked else where?

But, I know little about this. Can someone who knows more look at this?

### Point of contact on this issue
Who can we follow-up with if we have questions?
I @cmajel can verify but maybe update this with an engineer with relevant experience?

### Reproduction steps (if necessary)
Be as specific as possible
* Google `18f us tax court`, and click a case study URL
* Google `18f fbi crime data explorer`, and click a case study URL

**Billable?**

- [ ] Yes
- [x] No
If yes, tock code:

**Skills needed**

- [ ] Any human
- [ ] Design
- [ ] Content
- [x] Engineering
- [ ] Acquisition
- [ ] Product
- [ ] Other

**Timeline**
Does this need to happen in the next two weeks?

- [x] Yes*
- [ ] No

* Saying yes because seems like a bug

How much time do you anticipate this work taking?
* Less than two weeks

### Acceptance Criteria

- [ ] * Google `18f us tax court`, and click a case study URL
- [ ] * Google `18f fbi crime data explorer`, and click a case study URL

ex: As an 18F staff member who is going on parental leave, I can navigate to my
latest Tock reporting period and choose billing line #1810 by filtering by
number or "extended OOO"

**Comment by michelle-rago** [comment]
I requested reindexing of the updated pages on Google. This is pending. I'll report back.

**Comment by michelle-rago** [comment]
This work is still pending. Waiting on Google...

**Comment by michelle-rago** [comment]
I'm still trying to make progress on this. I've been requesting indexing page by page (which isn't working in every case). I'm wondering if we need to update how our [sitemap.xml file](https://18f.gsa.gov/sitemap.xml) is being generated. The latest version doesn't include the case study pages that were moved under /our-work/. 

@MelissaBraxton @alexbielen

**Comment by alexbielen** [comment]
@michelle-rago I created an issue to track sitemap.xml updates

**Comment by michelle-rago** [comment]
> @michelle-rago I created an issue to track sitemap.xml updates
I'm starting to think that the Google issue won't be fixed until the sitemap is updated. Google refuses to index https://18f.gsa.gov/our-work/forest-service/ no matter how nicely (or often) I ask.

**Comment by michelle-rago** [comment]
Yay, the Forest Service case study has finally been indexed.
